src/harness/testcases/testdata/wdb_2/CPI_Database-Public.csv

```diff
@@ -0,0 +1,3 @@
+cpiId,status,publicKeyIdentifier 
+ABCdef123,ACTIVE,
+123456,ACTIVE,https://winnf.memberclicks.net/mcdatafiles/receiptattach/winnf/12764702/8960734/Public_Key_Dummy_File.txt

```

Comment:
The CPI database file is provided by the organization with authority which will be WinnForum in this case. I think we shall keep what we have from the WinnForum. Please advise if you think otherwise